fixtures/external-durable-objects-app/tests/index.test.ts

```diff
@@ -28,7 +28,6 @@ describe(
 			a = await unstable_dev(path.join(__dirname, "../a/index.ts"), {
 				config: path.join(__dirname, "../a/wrangler.toml"),
 				experimental: {
-					fileBasedRegistry: true,
```

Comment:
Had to remove this to get this test suite to pass cc @penalosa

Reply:
Basically, it couldn't find MyDurableObject in the dev registry.

It was an obscure error because the test wants calls response.json() but Miniflare's EXTERNAL_SERVICE_WORKER_SCRIPT returns this text (non-json) response:
https://github.com/cloudflare/workers-sdk/blob/fixtures-x-dev-env/packages/wrangler/src/dev/miniflare.ts#L111

@penalosa and I agreed that since we're trying to default-on --x-dev-env before we default-on --x-registry that we should remove this flag in these tests for now and address independently